src/mono/wasm/runtime/binding_support.js

```diff
@@ -4,7 +4,7 @@
 var BindingSupportLib = {
 	$BINDING__postset: 'BINDING.export_functions (Module);',
 	$BINDING: {
-		BINDING_ASM: "[System.Runtime.InteropServices.JavaScript]System.Runtime.InteropServices.JavaScript.Runtime",
+		BINDING_ASM: "[System.Private.Runtime.InteropServices.JavaScript]System.Runtime.InteropServices.JavaScript.Runtime",
 		mono_wasm_object_registry: [],
```

Comment:
There appears to be some fallback code(??) a few lines down in the file that names the assembly directly.

Reply:
Do you mean here?  https://github.com/dotnet/runtime/blob/427f4f09cebdefc768909461af8f08625c2fb3c2/src/mono/wasm/runtime/binding_support.js#L78

Unless I'm mistaken, that's using the namespace, which is still the same.

Reply:
You are correct!